controller/controller_test.go

```diff
@@ -255,13 +256,25 @@ func (s *S) TestCreateRelease(c *C) {
 
 func (s *S) TestCreateFormation(c *C) {
 	for i, useName := range []bool{false, true} {
+		// First create an empty release and attempt to scale a nonexistent process type. Will fail.
 		release := s.createTestRelease(c, &ct.Release{})
 		app := s.createTestApp(c, &ct.App{Name: fmt.Sprintf("create-formation-%d", i)})
-
 		in := &ct.Formation{ReleaseID: release.ID, AppID: app.ID, Processes: map[string]int{"web": 1}}
 		if useName {
 			in.AppID = app.Name
 		}
+		err := s.c.PutFormation(in)
+		c.Assert(hh.IsValidationError(err), Equals, true)
```

Comment:
I don't set the code right now since that wasn't the pattern in other code I cribbed from. I could do that, however, if you prefer.